test/quantization/core/test_workflow_module.py

```diff
@@ -908,6 +908,18 @@ def test_quant_min_max_override(self):
         self.assertEqual(fq_module.activation_post_process.quant_min, 0)
         self.assertEqual(fq_module.activation_post_process.quant_max, 127)
 
+    @given(device=st.sampled_from(['cpu', 'cuda'] if torch.cuda.is_available() else ['cpu']))
+    def test_fused_moving_avg_obs_fake_quant(self, device):
+        try:
+            dtype = torch.bfloat16 if device == 'cuda' else torch.float32
```

Comment:
should we also test cuda with torch.float32? If I understand correctly this test doesn't do that right now